microsoft-edge/webview2/how-to/set-preview-channel.md

```diff
@@ -0,0 +1,290 @@
+---
+description: How to specify a Microsoft Edge preview channel to use, to test experimental APIs in a prerelease package.
+title: Switch to a preview channel to test upcoming APIs and features
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 07/20/2021
+ms.topic: how-to
+ms.prod: microsoft-edge
+ms.technology: webview
+keywords: IWebView2, IWebView2WebView, webview2, webview, win32 apps, win32, edge, ICoreWebView2, ICoreWebView2Host, browser control, edge html
+---
+# Switch to a preview channel to test upcoming APIs and features
+
+Updates of the WebView2 Evergreen Runtime often include new APIs and features.  Some of these updates could potentially break your WebView2 app.  To test the experimental APIs ahead of time and ensure your app's forward-compatibility, you should perform compatibility testing using a preview channel of Microsoft Edge, along with a prerelease version of the WebView2 SDK.
+
+
+## Combinations of the SDK and browser channel 
+
+Experimental WebView2 APIs are for use during development and testing of your WebView2 app, not for a production release of your WebView2 app.
+
+*   During development and testing of your WebView2 app, to use experimental WebView2 APIs, use the following combination:
+    *   A prerelease version of the WebView2 SDK.
+    *   A Microsoft Edge preview channel (Beta, Dev, or Canary).  Canary is recommended.
+
+*   For a production release of your WebView2 app, use the following combination, which doesn't provide experimental WebView2 APIs:
+    *   A release version of the WebView2 SDK.
+    *   The WebView2 Runtime.
+
+To develop a WebView2 application that uses experimental APIs, you need two components:
+*  A _prerelease_ version of the WebView2 SDK.  A release version of the SDK doesn't have experimental WebView2 APIs.
+*  A Microsoft Edge _preview_ channel (Beta, Dev, or Canary), also called a Microsoft Edge _Insiders_ channel.
+   *  The WebView2 Runtime doesn't support the WebView2 experimental APIs.
+   *  The Stable version of Microsoft Edge doesn't support WebView2.
+
+### Relation of the SDK and a channel or Runtime
```

Comment:
release-notes.md in PR for distribution.md expresses this section's info in the form of a Note: 
```
> [!NOTE]
> Make sure to re-compile your WebView2 app after updating the WebView2 SDK NuGet package.  The WebView team recommends that you use the Canary preview channel of Microsoft Edge when you develop using a prerelease version of the WebView2 SDK package, and use the Evergreen WebView2 Runtime when you use a release version of the WebView2 SDK package.  For more information, navigate to [Matching the Runtime version with the SDK version][Webview2ConceptsVersioningMatchingWebview2RuntimeVersions].
<!-- links -->
[Webview2ConceptsVersioningMatchingWebview2RuntimeVersions]: ./concepts/versioning.md#matching-the-runtime-version-with-the-sdk-version "Matching the Runtime version with the SDK version - Understand WebView2 SDK versions | Microsoft Docs"
```